app/store/fetchPlayersReducer.js

```diff
@@ -1,14 +1,12 @@
-import apiService from './services/ApiService'
+import apiService from '../services/ApiService'
```

Comment:
Do you think you could look into using absolute paths at some point? I couldn't get it working when I set up the project, but it would certainly tidy things up a little.